pig/src/main/java/org/apache/zeppelin/pig/PigInterpreter.java

```diff
@@ -58,6 +59,12 @@ public void open() {
     }
     try {
       pigServer = new PigServer(execType);
+      for (Map.Entry entry : getProperty().entrySet()) {
+        if (!entry.getKey().toString().startsWith("zeppelin.")) {
+          pigServer.getPigContext().getProperties().setProperty(entry.getKey().toString(),
+              entry.getValue().toString());
```

Comment:
is it better to set a variable as `pigServer.getPigContext().getProperties()` to reuse?